### PR TITLE
Add yaml file for prepobsaero task

### PR DESCRIPTION
**Pull request by ypwang19**
This PR adds the yaml template prep_viirs.yaml.j2 needed for running the VIIRS aerosol observation preparation task.

### CODE REVIEW

File: parm/aero/prep/prep_viirs.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,15 @@
+provider: VIIRSAOD
+window begin: {{ window_begin }}
```
Comment (CoryMartin-NOAA) [inline parm/aero/prep/prep_viirs.yaml.j2]: do we need ` | to_isotime ` here on these two lines?
`{{ window_begin | to_isotime }}`

Comment (ypwang19) [reply parm/aero/prep/prep_viirs.yaml.j2]: What's the purpose of` | to_isotime`? Now it works fine without it, but I can add it if needed.

Comment (CoryMartin-NOAA) [reply parm/aero/prep/prep_viirs.yaml.j2]: I think it should ensure that the date is formatted `YYYY-MM-DDTHH:MM:SSZ`

**Review by CoryMartin-NOAA — APPROVED** [review]
looks great thanks!

**Pull request merged**